muse_payment_adaptor/models.py

```diff
@@ -1,3 +1,16 @@
 from django.db import models
 
-# Create your models here.
+from core.models import HistoryBusinessModel
+from location.models import HealthFacility
+
+
+class HFBankInformation(HistoryBusinessModel):
+    health_facility = models.ForeignKey(HealthFacility, models.DO_NOTHING, db_column='HFId', blank=True, null=True)
```

Comment:
Can we make the FK based on UUID?

Reply:
@dragos-dobre I think we should leave the FK for now and then update all the IDs in one go. Updating the approach just in one case won't have any benefits.